src/collective/cover/content.py

```diff
@@ -128,6 +135,33 @@ def set_tile_data(self, id, **data):
         data_mgr = ITileDataManager(tile)
         data_mgr.set(data)
 
+    def get_referenced_objects(self):
+        """Get referenced objects from cover object.
+
+        :returns: a set of objects referenced
+        :rtype: set of objects
+        """
+        refs = set()
+        for tile_uuid in self.list_tiles():
+            tile = self.get_tile(tile_uuid)
+            uuid = tile.data.get('uuid', None)
+            if uuid is not None:
```

Comment:
@rodfersou I think you should move these below on a final `else` statement or you are going to insert an invalid UUID on the list: both List and RichText tiles have a uuid field.